Source/WebCore/dom/Document.cpp

```diff
@@ -6294,7 +6295,7 @@ Ref<HTMLCollection> Document::images()
 
 Ref<HTMLCollection> Document::applets()
 {
-    return ensureCachedCollection<DocEmpty>();
+    return ensureRareData().ensureNodeLists().addCachedCollection<EmptyHTMLCollection>(*this, DocEmpty);
```

Comment:
I think it would be nice to move `DocEmpty` to the ASSERT_NOT_REACHED() section (with `DocAll`) inside
`GenericCachedHTMLCollection<traversalType>::elementMatches(Element&` instead of making it look like GenericCachedHTMLCollection is dealing with this type.

Reply:
Oh, right, I forgot to do that.